src/utils/calendarItems.js

```diff
@@ -0,0 +1,45 @@
+import { subMonths } from 'date-fns';
+import range from 'lodash/range';
+
+import calendarMatrix from './calendarMatrix';
+import { formatDayKey } from './dateUtils';
+
+function addWeekItem(week, weekIndex, monthIndex, items, todayKey) {
+  const weekKey = `week-${monthIndex}-${weekIndex}`;
+  const current = week.map((day) => day.key).includes(todayKey);
```

Comment:
Creo que aquí conviene usar [`some`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some):

```suggestion
  const current = week.some((day) => day.key === todayKey);
```

Así paras de buscar apenas lo encuentres y en estricto rigor, iteras menos

Reply:
Bueena buena!